packages/block-editor/src/components/inserter/search-results.js

```diff
@@ -199,6 +202,15 @@ function InserterSearchResults( {
 		</InserterPanel>
 	);
 
+	if ( isZoomOutMode ) {
+		return (
+			<InserterListbox>
+				{ ! hasItems && <InserterNoResults /> }
+				{ patternsUI }
```

Comment:
Since we now only render patterns, this should be only about patterns.
```suggestion
				{ patternsUI || <InserterNoResults /> }
```